sync/device.ts

```diff
@@ -24,7 +24,7 @@ export async function getSupervisorContainer(
 ): Promise<Docker.ContainerInfo> {
 	// First get the supervisor container id
 	const containers = await docker.listContainers({
-		filters: { name: ['resin_supervisor'] },
+		filters: { name: ['balena-supervisor'] },
```

Comment:
@pipex we agreed to go for `balena_supervisor` as there was already an open PR https://github.com/balena-os/meta-balena/pull/2171

Reply:
Yes, I only added this for the benefit of testing. I'll change it before we merge